crime-means-assessment/src/main/java/uk/gov/justice/laa/crime/meansassessment/builder/CreateInitialAssessmentBuilder.java

```diff
@@ -17,7 +17,7 @@ public ApiCreateAssessment build(InitialMeansAssessmentDTO initialMeansAssessmen
                 .withInitialAscrId(initialMeansAssessmentDTO.getAssessmentCriteria().getId())
                 .withCmuId(initialMeansAssessmentDTO.getMeansAssessment().getCmuId())
                 .withFassInitStatus(initialMeansAssessmentDTO.getInitStatus().getStatus())
-                .withInitialAssessmentDate(initialMeansAssessmentDTO.getMeansAssessment().getAssessmentDate())
+                .withInitialAssessmentDate(initialMeansAssessmentDTO.getMeansAssessment().getInitialAssessmentDate())
```

Comment:
As discussed, we need to ensure that null is still being sent across for the result and resultReason fields if the assessment result is NONE.